### ISSUE TITLE
count_files should allow ls -1a | wc -w

### ISSUE THREAD

**Issue by PeterDavidCarter** [jarv/cmdchallenge#92]
Note: if you are having trouble with a challenge you can see solutions here -> https://github.com/jarv/cmdchallenge/blob/master/challenges.yaml
Search for the challenge and look for the `example:` to see one way of doing it.

If you would like to help improve the wording of a challenge description, please consider sending a pull request against https://github.com/jarv/cmdchallenge/blob/master/challenges.yaml

## Summary

As the challenge text states:

> # Count the number of files in the current
> # working directory. Print the number of
> # files as a single integer.

Technically the correct answer is: `ls -1a | wc -w` and not `ls -1 | wc -w`. The question should be amended to note that the output should exclude hidden files, or the `ls -1a | wc -w` answer should be accepted as valid.

## Steps to reproduce
If applicable please provide a direct link to the challenge, example: https://cmdchallenge.com/#/sum_all_numbers

Try `ls -1a | wc -w`.

**Comment by 0ki** [comment]
This is invalid.

1) "4" is not the number of files
2) . and .. are not files
3) what about files with whitespace?

**Comment by PeterDavidCarter** [comment]
Ah, true. The correct answer then would be `ls -1A | wc -l` or `find -type f -ls | wc -l`...

**Comment by jarv** [comment]
Yeah you are right that `ls -1A | wc -l` is the better answer, maybe throw in some dot files? Though I get plenty of complaints about this problem without making it even more "tricky".
